confidential-ledger-demo/server/confidential-setup.py

```diff
@@ -19,11 +22,14 @@
 
 #Set tenant ID from .env file
 
-TENANT_ID = os.getenv('AZURE_TENANT_ID')
-os.environ["AZURE_TENANT_ID"] = TENANT_ID
+clientId = os.getenv("CL_APP_ID")
+clientSecret = os.getenv("CL_CLIENT_SC")
+tenantId = os.getenv("AZURE_TENANT_ID")
```

Comment:
Duplicate file

Reply:
Setup-py isn't a duplcicate, it just gives examples of how to use the sdk to create and manage a CL. Do you think this would be better in an md instead?

Reply:
Either an MD or an example directory

Reply:
opened issue #107 to do this